src/library/ksTable/KsTable.jsx

```diff
@@ -100,6 +100,11 @@ const KsTable = ({
 
   const rowToggle = selectable ? () => {} : false;
   const onSelectedRowsChange = selectable ? () => {} : undefined;
+  let api;
+
+  React.useEffect(() => {
+    if (api) api.updateTable({ columns: columns });
```

Comment:
Can't you do it in this way?
```suggestion
    if (api) api.updateTable({ columns });
```